frontend/src/components/Pages/Media/Media.css

```diff
@@ -20,45 +24,72 @@ a.nostyle:link {
   cursor: auto;
 }
 .news_description
-{
-  width: 50%;
-  float: left;
-  text-align: center;
-  font-size: larger;
-  font-weight: 700;
-  margin-top: 20px;
-  margin-left: 6px;
-  cursor: pointer
+{ text-align: center;
+  cursor: pointer;
+  line-height: 0.5;
+}
+
+.mediaBanner {
+  background-image: url('../../../assests/pageBanners/image4.jpeg');
 }
 
-.date_heading {
-  font-size: 22px;
+.newsCollection{
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  font-size: 20px;
 }
 
-.image_news {
-   width: 350px !important;
-   height:190px !important
+.newsBoxTop{
+  width: 99%;
+  height: 30%;
+  display: flex;
+  flex-flow: row nowrap;
+  line-height: 1.13;
 }
 
-@media all and (min-width:0px) and (max-width: 480px) {
-  /* put your css styles in here */
-  .newsbox {
-    height: 400px;
-    width: 400px;
-    margin: 2px 22px;
-    border: 2px solid wheat;
+.newsBoxTopImage{
+  width:25%;
+  height: 80%;
+  padding-left: 1%;
+  padding-top: 1%;
 }
+
+.newsBoxTopImage img{
+  height: 100%;
+  width: 100%;
+}
+.newsBoxTopContent{
+  width: 75%;
+  display: flex;
+  vertical-align: middle;
+  justify-content: center;
+  flex-flow: column nowrap;
+  padding: 4%;
 }
 
-@media only screen and (min-device-width : 768px) and (max-device-width : 1024px) {
```

Comment:
A basic responsive web app guideline, media queries should use in order to make app responsive not the Percentage.

Reply:
added media queries to for the news box class. but still using fluid for everything else.

Reply:
also added extra margin to avoid overflow of hindi text.